static/docs/use-cases/how-to-store-files-in-dvc.md

```diff
@@ -0,0 +1,129 @@
+# How to store files in DVC
+
+DVC allows storing and versioning files with git, without checking the file
+contents into git. It is useful when dealing with files that are too large
+for git to handle. DVC stores information about your data file in a special
+.dvc file, that has a description of a file that can be used to version it.
+DVC supports various types of remote locations for your cache files and
+allows you to easily store and share your data alongside your code.
+
+Let's say you already have a project that uses a bunch of images that are
+stored in `assets` directory.
+
+```sh
+    $ ls assets
+    favicon-16x16.png favicon-32x32.png favicon.ico social-share.png
+```
+
+To start using dvc we need to first initialize it in your repository:
+
+```sh
+    $ dvc init
+```
+
+DVC will create a `.dvc` directory that will store special files and also
```

Comment:
I'd use present simple tense here. "DVC stores special files in a `.dvc` directory where `.dvc/cache` is used for ".

Btw... do we need to go to the details from the very beggining? Why don't we move this part in the last paragraph - "how DVC works?" - just asking

Reply:
Ok.